### ISSUE TITLE
[staging] Sync BRD and EKS settings

### ISSUE THREAD

**Issue by rmtolmach** [department-of-veterans-affairs/va.gov-team#52880]
I went through [staging-settings.local.yml.j2](https://github.com/department-of-veterans-affairs/devops/blob/master/ansible/deployment/config/vets-api/staging-settings.local.yml.j2) and found all of the `.key`, `.pem` values under `/etc/pki/tls/certs` and `/srv/vets-api/secret`. I copied each of the paths and ran a `cat` command on a vets-api server EC2 instance and compared the output to running the same `cat` command on a vets-api-web instance in ArgoCD. These are the values that had some mismatch:

- [x] `/srv/vets-api/secret/lighthouse_fast_track_api.key` - no such file or dir in Argo. IT'S `.keys` in EKS 🤦‍♀️  fixing in https://github.com/department-of-veterans-affairs/vsp-infra-application-manifests/pull/1389 but this shouldn't be part of the problem.
- [x] `/srv/vets-api/secret/gibft-sfdc-signing.key` FIXED IN PARAMETER STORE!
- [x] `/srv/vets-api/secret/logingov_oauth_pub.pem` - it matches but its contents are just a little public key. Maybe this is fine??
    - [x] same with `idme_oauth_pub.pem`
    - [x] same with `/srv/vets-api/secret/production/dgi-int.crt`
- [x] `/srv/vets-api/secret/va_mobile.pem` - DOESN’T match. FIXED [in AWS](https://console.amazonaws-us-gov.com/systems-manager/parameters/devops/certificates/va_mobile.pem/description?region=us-gov-west-1&tab=Table#list_parameter_filters=Name:Contains:va_mobile) by putting the BRD value in param store
- [x] `health-quest.lighthouse.key` - DOESN’T match and no newline in ec2. [FIXED in AWS](https://console.amazonaws-us-gov.com/systems-manager/parameters/dsva-vagov/vets-api/staging/healthquest/key/history?region=us-gov-west-1&tab=Table#list_parameter_filters=Name:Contains:%2Fdsva-vagov%2Fvets-api%2Fstaging%2Fhealthquest%2Fkey) so EKS matches the value in BRD. New line at end added.
- [ ] `/srv/vets-api/secret/vic-signing.key` - doens’t match. no newline for Argo instance
- [x] `check-in.lorota.v1.key` matches, but needs a new line at the end of the key. FIXED it [here in AWS](https://console.amazonaws-us-gov.com/systems-manager/parameters/dsva-vagov/vets-api/staging/check_in/lorota_key/description?region=us-gov-west-1&tab=Table#list_parameter_filters=Name:Contains:lorota)
- [x] `production/dgi-int.crt` needs newline at tend. FIXED it [here in AWS](https://console.amazonaws-us-gov.com/systems-manager/parameters/devops/certificates/dgi-int.crt/description?region=us-gov-west-1&tab=Table#list_parameter_filters=Name:Contains:dgi-int)
- [x] `/etc/pki/ca-trust/source/anchors/va.pem` (NO SUCH FILE OR DIR in eks, but maybe this doesn’t matter b/c of the file path??)
- [x] `/etc/pki/tls/private/vetsgov-mvi.key` (did not match. replaced value found in an ec2 instance with what was in paramstore)

**Comment by rmtolmach** [comment]
confirmed:
✅ /srv/vets-api/secret/lighthouse_fast_track_api.key
✅ /srv/vets-api/secret/gibft-sfdc-signing.key
✅ little baby keys are fine
✅ /srv/vets-api/secret/va_mobile.pem
✅ /srv/vets-api/secret/health-quest.lighthouse.key
❌ /srv/vets-api/secret/vic-signing.key is FUNKY. They don't match and there is no `-----BEGIN RSA PRIVATE KEY-----` on either
✅ /srv/vets-api/secret/check-in.lorota.v1.key
✅ /srv/vets-api/secret/production/dgi-int.crt
✅ in BRD: /etc/pki/ca-trust/source/anchors/va.pem in EKS: /etc/pki/tls/certs/anchors-va.pem

**Comment by laineymajor** [comment]
Complete

**Comment by laineymajor** [comment]
@rmtolmach we were talking with Bill about everything that's happened with staging.. he would like to get involved if there's somewhere we think he could help. 

We were thinking it could be beneficial for the two of you to review this manual, tedious process you went through... he hopes he can brainstorm with us ways to make this better in prep for prod.

We can talk about it in standup, just giving you a heads-up

**Comment by rmtolmach** [comment]
There ended up being some problems with this one:
 
> `/etc/pki/tls/private/vetsgov-mvi.key` (did not match. replaced value found in an ec2 instance with what was in paramstore)

I changed the wrong value in param store. I changed `/dsva-vagov/fwdproxy/int/mvi_key` instead of `/dsva-vagov/fwdproxy/qa/mvi_key`. This was resulting in the fwdproxy jenkins build failing. ([Slack thread](https://dsva.slack.com/archives/C04CYC4LMU6/p1675207484794129) for all the nitty, gritty details.) I changed the first one back to what it was before I messed with it and left qa as-is.

I then went back and did a thorough check of the mvi keys in https://github.com/department-of-veterans-affairs/vsp-infra-application-manifests/pull/1459. So we're good now! 🎉
